src/api/lib/tasks/databases.rake

```diff
@@ -83,6 +83,20 @@ namespace :db do
       end
       File.open("#{Rails.root}/db/structure.sql", "w+") { |f| f << new_structure }
     end
+
+    desc "Verify that structure.sql in git is up to date"
+    task verify: :environment do
+      puts "Running rails db:migrate"
+      Rake::Task["db:migrate"].invoke
```

Comment:
> And the db:create you propose would also create the tables, right?

No it only creates the database, not the tables.

> Running the migrations updates the structure.sql even when there is no migration ins the queue.

Ok if thats the case then I think this is fine :+1: